js/app.js

```diff
@@ -60,6 +62,14 @@ Calculator.prototype.pow = function (num1, num2) {
   });
 };
 const calc = new Calculator();
+
+const operationMap = {
+  '+': calc.add.bind(calc),
+  '-': calc.subtract.bind(calc),
+  '*': calc.multiply.bind(calc),
+  '/': calc.divide.bind(calc),
+  '^': calc.pow.bind(calc),
+};
```

Comment:
Jeszcze lepiej by było, gdyby ta lista była zdefiniowana wew. konstruktora np. wrzucić ją do `this.actions` - obecnie mamy niepotrzebne powtórzenie tego elementu w tych miejscach.